content/posts/getting-started/prepare-site/index.md

```diff
@@ -310,203 +310,17 @@ badges:
 
 Put the `resume.pdf` file in `/static/files` directory of your repository. You can find the `about.yaml` file used in the example site from [here](https://github.com/hugo-toha/hugo-toha.github.io/blob/source/data/en/sections/about.yaml).
 
-###### Skills Section
+###### Add other sections
```

Comment:
Now, it is much better. Thank you @BernatBC. You can simplify the list with just links. No other details is necessary. For example:

```markdown
In order to keep this post short, we have splitted it in different posts. Here, is the list of posts that will show you how to configure the other sections:

- [Configuring About Section](/posts/configuration/sections/about/).
- [Configuring Education Section](/posts/configuration/sections/education/).
- [Configuring Experiences Section](/posts/configuration/sections/experiences/).
- [Configuring Projects Section](/posts/configuration/sections/projects/).
- [Configuring Recent Posts Section](/posts/configuration/sections/recent-posts/).
- [Configuring Achievements Section](/posts/configuration/sections/achievements/).
- [Configuring Skills Section](/posts/configuration/sections/skills/).
```